aether-kernel/aether/kernel/api/tests/test_views.py

```diff
@@ -928,3 +930,22 @@ def test_submission_validate(self):
         response_data = response.json()
 
         self.assertIn('is not a valid UUID', response_data)
+
+    def test_delete_by_filters(self):
+        # Generate projects.
+        for _ in range(random.randint(5, 10)):
+            generate_project()
+        entity_count = models.Entity.objects.count()
+        mapping = models.Mapping.objects.first()
+        filtered_count = models.Entity.objects.filter(mapping=mapping.pk).count()
+        url = f'{reverse("entity-filtered-delete")}?mapping={str(mapping.id)}'
+        response = self.client.delete(url)
+        self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
+        self.assertEqual(
+            models.Entity.objects.count(),
+            (entity_count - filtered_count)
+        )
+
```

Comment:
could you add a test with no query string and check that everything is deleted? :scream: